include/af/graphics.h

```diff
@@ -43,6 +45,9 @@ class AFAPI Window {
 
         void initWindow(const int width, const int height, const char* const title);
 
+        Window(const Window&);                 // Prevent copy-construction
```

Comment:
Pradeep this may cause missing symbol errors I think. Don't we need to have " {} "  as well?

@umar456 I am not entirely sure about this can you take a look ?